test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs

```diff
@@ -39,7 +39,7 @@ public class GivenThatIWantToShowHelpForDotnetCommand : TestBase
   msbuild          Runs Microsoft Build Engine (MSBuild).
   vstest           Runs Microsoft Test Execution Command Line Tool.
   store            Stores the specified assemblies in the runtime store.
-  install          Add item to development environment.
+  install          Installs an item into the development environment.
```

Comment:
This was a long PR. If I didn't comment on something, point it out as I think I left comments where I focused. If I missed something let me know.